speech/google/cloud/speech/_gax.py

```diff
@@ -104,9 +104,9 @@ def async_recognize(self, sample, language_code=None,
         audio = RecognitionAudio(content=sample.content,
                                  uri=sample.source_uri)
         api = self._gapic_api
-        response = api.async_recognize(config=config, audio=audio)
+        operation_future = api.async_recognize(config=config, audio=audio)
 
-        return Operation.from_pb(response, self)
+        return Operation.from_pb(operation_future.last_operation_data(), self)
```

Comment:
Hmm... interesting point @daspecster. We might want to leave this as-is for now, until we can provide an `_OperationFuture` wrapper for HTTP/1.1 in `google-gax`, unless you can @lukesneeringer can think of a better solution. We don't want to break the code re-use you have between the two transports.